.env.example

```
@@ -57,4 +57,8 @@ STATSIG_API_KEY=
 NODE_ENV=development
 PORT=5000
 CORS_ORIGINS=http://localhost:3000,http://localhost:4000,http://localhost:3001
-CONTRIBUTOR_APP_URL=http://localhost:4000
\ No newline at end of file
+CONTRIBUTOR_APP_URL=http://localhost:4000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CONTRIBUTOR_APP_URL key should go before the CORS_ORIGINS key

(UnorderedKey)